bloom/commands/release.py

```diff
@@ -460,6 +460,7 @@ def convert_unicode_dict_to_str(d):
     track_dict = get_tracks_dict_raw()['tracks'][track]
     last_version = track_dict['last_version']
     release_inc = track_dict['release_inc']
+    version = '{0}-{1}'.format(last_version, release_inc)
```

Comment:
I can't see where this value gets used...do you mean to modify `repo['version']` below as well?

Reply:
It gets used on https://github.com/ros-infrastructure/bloom/blob/be9d9b8763e4fcc98dba9b9bb7cafe0ed22a375a/bloom/commands/release.py#L638.

You're right that I can probably use this variable to set that field.

Reply:
1a92538 uses this variable to set repo['version'].